src/System.Net.Utilities/tests/FunctionalTests/TestSettings.cs

```diff
@@ -10,37 +10,41 @@ namespace System.Net.Utilities.Tests
     internal static class TestSettings
     {
         public static readonly string LocalHost = "localhost";
-        public const int PingTimeout = 200;
+        public const int PingTimeout = 1000;
 
         public const string PayloadAsString = "'Post hoc ergo propter hoc'. 'After it, therefore because of it'. It means one thing follows the other, therefore it was caused by the other. But it's not always true. In fact it's hardly ever true.";
         public static readonly byte[] PayloadAsBytes = Encoding.UTF8.GetBytes(TestSettings.PayloadAsString);
 
         public static Task<IPAddress> GetLocalIPAddress()
         {
-            return ResolveHost(LocalHost, AddressFamily);
+            return ResolveHost(LocalHost);
         }
 
-        public static AddressFamily AddressFamily
-        {
-            get
-            {
-                return AddressFamily.InterNetworkV6;
-            }
-        }
-
-        private static async Task<IPAddress> ResolveHost(string host, AddressFamily family)
+        private static async Task<IPAddress> ResolveHost(string host)
         {
             IPHostEntry hostEntry = await Dns.GetHostEntryAsync(host);
+            IPAddress ret = null;
 
             foreach (IPAddress address in hostEntry.AddressList)
             {
-                if (address.AddressFamily == family)
+                if (address.AddressFamily == AddressFamily.InterNetworkV6)
                 {
-                    return address;
+                    ret = address;
```

Comment:
That's probably better. Before, it had parameters for the AddressFamily, but we were only using it to find the local IPv6 address. That didn't work when the machine had no IPv6 addresses, so I just added a fallback for an IPv4 address. I'll just put this into the GetLocalIPAddress method now. Like I mentioned above, I am planning to rework a bit how these tests are structured, so that we can test both IPv4 and IPv6 localhost pings if the machine supports it, rather than just IPv6 and falling back to IPv4 if necessary (which is the state of this PR).